src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,60 @@
+.logo-link {
+  line-height: 0;
+  margin: 10px 50px;
+}
+
+.nav {
+  padding: 19px 0;
+  margin-right: 50px;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.